specs/integration/search/CreateResourceWidget.js

```diff
@@ -0,0 +1,10 @@
+titleField: "input#title",
+openFileUploaderLink: '.resource-upload-dropzone input[type="file"]',
+
+fill: function fill(name) {
+    return  this.setTitleField(name)();
+},
+
+waitForUpload: function waitForUpload() {
+    setTimeout(function() {}, 7000); // TODO: fragile but improve your APM ;)
```

Comment:
Oh it explains everything, thanks for the suggestions, I'll go with the assertion on upload completed.